modules/storages/app/views/storages/admin/storages/update.turbo_stream.erb

```diff
@@ -20,6 +20,4 @@
   <%= full_breadcrumbs %>
 <% end %>
 
-<%= turbo_stream.update :storage_general_info_section do %>
-  <%= render(::Storages::Admin::GeneralInfoComponent.new(@storage)) %>
-<% end %>
+<%= render(::Storages::Admin::GeneralInfoComponent.new(@storage)) %>
```

Comment:
Small 🐛  here- need to use `update_via_turbo_stream` so there is an actual `turbo.update`

Reply:
https://github.com/opf/openproject/pull/15922/commits/dc75b7f4d746a5237c843f2b5c50fa73f2a55873